router/router.go

```diff
@@ -304,6 +304,7 @@ func SupportCORS(handler http.Handler) http.Handler {
 		AllowOriginFunc: func(string) bool {
 			return true
 		},
+		AllowedOrigins: []string{"*"},
```

Comment:
We actually can't do this. It will break in browsers. See: https://github.com/rs/cors/issues/55